src/lib/progress-bar/progress-bar.scss

```diff
@@ -1,12 +1,14 @@
 @import '../core/style/variables';
 @import '../core/style/vendor-prefixes';
+@import '../core/style/noop-animation';
 
 $mat-progress-bar-height: 5px !default;
 $mat-progress-bar-full-animation-duration: 2000ms !default;
 $mat-progress-bar-piece-animation-duration: 250ms !default;
 
 
 .mat-progress-bar {
+  @include noop-animation();
```

Comment:
I'm not a fan of the way the mixin is peppered throughout the file in so many places. Let's take a look next week to see what it would look like with the noop animations code in one place.